docs/marketplace/add_to_marketplace/add_item_by_type/add_custom_resource.md

```diff
@@ -0,0 +1,256 @@
+        }
+        }
+    },
+    "annotations": {
+        "type": "array",
+        "items": {
+        "type": "object",
+        "properties": {
+            "name": {
+                "type": "string"
+            },
+            "value": {
+                "type": "string"
+            }
+        }
+        }
+    }
+    }
+}
+```
+
+</p>
+</details>
+
+A simple example in YAML of a custom resource that follows the schema is the following:
+
+<details>
+<summary>Kube Green</summary>
+<p>
+
+```yaml
+name: sleepInfo
+  meta:
+    apiVersion: kube-green.com/v1alpha1
+    kind: SleepInfo
+  spec:
+    sleepAt: "20:00"
+    timeZone: Europe/Rome
+    weekdays: "1-5"
+```
+
+</p>
+</details>
+
+## Defined the new Custom Resource
+
+::: info
+In the paragraphs to follow, wwe are going to use the `miactl` tool to show how to create a new custom resource.
+:::
+
+First of all, we need to create a new file that includes the marketplace information and the custom resource that we want to add.
+
+```yaml
+name: Sleep Info
+itemId: sleep-info
+tenantId: my-company
+version:
+  name: 1.0.0
+  releaseNote: "First release"
+description: A Custom Resource dedicated to manage the shutdown time of the application
+type: custom-resource
+documentation:
+  type: externalLink
+  url: https://github.com/my-sidecar/docs
+resources:
+  name: sleep-info
+  meta:
+    apiVersion: kube-green.com/v1alpha1
+    kind: SleepInfo
+  spec:
+    sleepAt: "20:00"
+    timeZone: Europe/Rome
+    weekdays: "1-5"
+```
+
+Please note that:
+
+ -the `type: custom-resource` is required to specify that this resource is a custom resource
+- the `version` property is included, to define this item as a version `1.0.0` with a specific release note: versions are not mandatory but highly suggested to avoid to overwrite previous versions
+
+### Publish a new custom resource via `miactl`
+
+This section describes the steps required to publish a sidecar resource to the Marketplace via [miactl](/cli/miactl/10_overview.md), the Command Line Interface by Mia-Platform designed to interact with the Mia-Platform Console. Make sure to [configure miactl](/cli/miactl/20_setup.md) before proceeding with the next steps.
+
+To upload the resource to the marketplace, run the following command, specifying the file name and your Company (tenant) id:
+
+```bash
+> miactl marketplace apply -f ./my-custom-resource.yaml --company-id my-company
+```
+
+You should receive a success response similar to this one:
+
+```bash
+1 of 1 items have been successfully applied:
+
+  OBJECT ID                 ITEM ID             STATUS   
+
+  66423781fdd3d6dd3ca62b7b  my-custom-resource  Created 
+```
+
+You just created your custom resourced, which is now available on your `Internal Company Marketplace`.
+
+### Update the custom resource via `miactl`
+
+A versioned custom resource can be updated only if changes apply to the following fields:
+
+- `name`
+- `description`
+- `imageUrl`
+- `supportedByImageUrl`
```

Comment:
these fields can't be managed by final users since they are restricted. Please verify the list with @danielemarostica

Reply:
Check the "Create your Company Marketplace" page, it should be properly updated there.